admin_manual/configuration/database/db_conversion.rst

```diff
@@ -8,6 +8,49 @@ If you have an existing ownCloud installation which uses SQLite, and you want to
 
 .. note:: ownCloud Enterprise edition does not support SQLite.
 
+
+Preparation
+-----------
+
+Add to your ownCloud config.php
+
+::
+
+	'mysql.utf8mb4' => true,
+
+Add or adjust in your /etc/mysql/mariadb.conf.d/50-server.cnf 
+
+..note::
+
+		You can do the same for mysql by replacing mariadb.conf.d/50-server.cnf with mysql.conf.d/mysqld.cnf
+
+
+::
+
+	key_buffer_size         = 32M
+	table_cache            = 400
+	query_cache_size        = 128M
+
+	#in InnoDB:
+	innodb_flush_method=O_DIRECT
+	innodb_flush_log_at_trx_commit=1
+	innodb_log_file_size=256Mser
```

Comment:
Did not catch that earlier...
``innodb_log_file_size=256Mser`` --> ``innodb_log_file_size=256M``